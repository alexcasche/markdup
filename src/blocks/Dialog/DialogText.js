import styled from 'styled-components'
import { tint } from 'polished'

const DialogText = styled.p`
  margin: 0;
  padding: 20px 20px 30px;
  text-align: left;
  color: ${props => tint(0.85, props.theme.colors.grey)};
  font-size: 17px;
  font-weight: 500;
  letter-spacing: 1px;
  line-height: 1.75;
  svg {
    color: ${props => props.theme.colors.green};
    margin-right: 10px;
  }
`

export default DialogText
