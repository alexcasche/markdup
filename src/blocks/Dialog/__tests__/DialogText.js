import DialogText from '../DialogText'

describe('DialogText Element', () => {
  it('renders correctly', () => {
    const tree = render(<DialogText />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
