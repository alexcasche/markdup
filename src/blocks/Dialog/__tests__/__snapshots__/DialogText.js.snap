// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DialogText Element renders correctly 1`] = `
.c0 {
  margin: 0;
  padding: 20px 20px 30px;
  text-align: left;
  color: #b7b9be;
  font-size: 17px;
  font-weight: 500;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
  line-height: 1.75;
}

.c0 svg {
  color: #34F96E;
  margin-right: 10px;
}

<p
  className="c0"
/>
`;
